src/main/java/eu/yals/exception/error/UserMessageGenerator.java

```diff
@@ -0,0 +1,22 @@
+package eu.yals.exception.error;
+
+import java.util.HashMap;
+
+import static eu.yals.constants.App.NO_STATUS;
+
+public class UserMessageGenerator {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'UserMessageGenerator' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5392475)